server/py/services/api/api/endpoints/operations.py

```diff
@@ -194,26 +194,31 @@ def _create_refresh_smtp_configuration_background_task(
 
 
 async def _perform_refresh_smtp(session: str):
-    iguazio_client_instance = iguazio_client.Client()
-    try:
-        returned_smtp_configuration = iguazio_client_instance.get_smtp_configuration(
-            session
-        )
-    except mlrun.errors.MLRunInternalServerError as exc:
-        logger.warning(
-            "Failed to get SMTP configuration from Iguazio",
-            exc=mlrun.errors.err_to_str(exc),
-        )
-        raise
-
-    updated_params = {
-        "server_host": returned_smtp_configuration.host,
-        "server_port": str(returned_smtp_configuration.port),
-        "sender_address": returned_smtp_configuration.sender_address,
-        "username": returned_smtp_configuration.auth_username,
-        "password": returned_smtp_configuration.auth_password,
-    }
-    _store_mail_notifications_default_params_to_secret(updated_params)
+    # if running in iguazio, get the SMTP configuration from iguazio and store it in k8s secret store
+    if mlrun.mlconf.iguazio_api_url:
+        # TODO: Fix for iguazio v4
```

Comment:
You can force it to not run for IG4
```suggestion
    if mlrun.mlconf.iguazio_api_url and not mlrun.mlconf.is_iguazio_v4_mode():
        # TODO: Fix for iguazio v4
```